src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/DiaryOverviewTableViewController.swift

```diff
@@ -35,41 +42,124 @@ class DiaryOverviewTableViewController: UITableViewController {
 
 	override func viewDidLoad() {
 		super.viewDidLoad()
+
+		setupTableView()
+		navigationItem.largeTitleDisplayMode = .always
+
+		navigationItem.title = AppStrings.ContactDiary.Overview.title
+		
+		let moreImage = UIImage(named: "Icons_More_Circle")
+		let rightBarButton = UIBarButtonItem(image: moreImage, style: .plain, target: self, action: #selector(onMore))
+		rightBarButton.tintColor = .enaColor(for: .tint)
+		self.navigationItem.setRightBarButton(rightBarButton, animated: false)
 	}
 
 	// MARK: - Protocol UITableViewDataSource
 
 	override func numberOfSections(in tableView: UITableView) -> Int {
-		// #warning Incomplete implementation, return the number of sections
-		return 0
+		return 2
 	}
 
 	override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-		// #warning Incomplete implementation, return the number of rows
-		return 0
+		switch section {
+		case 0:
+			return 1
+		case 1:
+			return diaryService.days.count
+		default:
+			fatalError("Invalid section")
+		}
 	}
 
-//	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-//		let cell = tableView.dequeueReusableCell(withIdentifier: "reuseIdentifier", for: indexPath)
-//
-//		// Configure the cell...
-//
-//		return cell
-//	}
+	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		switch indexPath.section {
+		case 0:
+			return descriptionCell(at: indexPath)
+		case 1:
+			return dayCell(at: indexPath)
+		default:
+			fatalError("Invalid section")
+		}
+	}
 
 	// MARK: - Protocol UITableViewDelegate
 
 	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-
+		onCellSelection(diaryService.days[indexPath.row])
 	}
 
 	// MARK: - Private
 
-	private let viewModel: DiaryOverviewViewModel
+	private let diaryService: DiaryService
 	private let onCellSelection: (DiaryDay) -> Void
 	private let onInfoButtonTap: () -> Void
 	private let onExportButtonTap: () -> Void
 	private let onEditContactPersonsButtonTap: () -> Void
 	private let onEditLocationsButtonTap: () -> Void
 
+	private var subscriptions = [AnyCancellable]()
+
+	private func setupTableView() {
+		tableView.register(
+			UINib(nibName: String(describing: DiaryOverviewDescriptionTableViewCell.self), bundle: nil),
+			forCellReuseIdentifier: String(describing: DiaryOverviewDescriptionTableViewCell.self)
+		)
+
+		tableView.register(
+			UINib(nibName: String(describing: DiaryOverviewDayTableViewCell.self), bundle: nil),
+			forCellReuseIdentifier: String(describing: DiaryOverviewDayTableViewCell.self)
+		)
+
+		tableView.separatorStyle = .none
+		tableView.rowHeight = UITableView.automaticDimension
+		tableView.estimatedRowHeight = 60
+	}
+
+	private func descriptionCell(at indexPath: IndexPath) -> UITableViewCell {
```

Comment:
```suggestion
	private func makeDescriptionCell(at indexPath: IndexPath) -> UITableViewCell {
```

Reply:
I was basically extracting code form the `cellForRowAt` func, and I think the naming without `make` is more consistent with the UITableViewDataSource protocol.